webapp/src/types/mattermost-webapp/index.d.ts

```diff
@@ -1,5 +1,22 @@
+import {GenericAction} from 'mattermost-redux/types/actions';
+import {Channel, ChannelMembership} from 'mattermost-redux/types/channels';
+import React from 'react';
+import {Reducer} from 'redux';
+
 export interface PluginRegistry {
-    registerPostTypeComponent(typeName: string, component: React.ElementType)
+    registerPostTypeComponent(typeName: string, component: React.ElementType);
```

Comment:
I guess this file has more information than the strictly needed because it is a copy past, but I think there is nothing wrong with this.